dandiapi/api/mail.py

```diff
@@ -66,3 +76,44 @@ def send_ownership_change_emails(dandiset, removed_owners, added_owners):
     messages += [build_added_message(dandiset, added_owner) for added_owner in added_owners]
     with mail.get_connection() as connection:
         connection.send_messages(messages)
+
+
+# Email sent to the DANDI list when a new user logs in for the first time
+
+
+def registered_subject(sociallogin):
+    return f'DANDI: New user registered: {sociallogin.user.email}'
+
+
+def registered_message(sociallogin):
+    return (
+        f'A new user has logged in to the DANDI application: {sociallogin.user.email}. '
+        f'Their GitHub login is {sociallogin.account.extra_data["login"]}.'
+    )
+
+
+def registered_html_message(sociallogin):
+    return (
+        f'A new user has logged in to the DANDI application: {sociallogin.user.email}. '
+        f'Their GitHub login is {sociallogin.account.extra_data["login"]}.'
+    )
```

Comment:
```
<p>Dear {sociallogin.account.extra_data[&quot;name&quot;]} (Github ID: {sociallogin.account.extra_data[&quot;login&quot;]}),</p>
<p>Welcome to DANDI. </p>
<p>You are now registered on the DANDI archive. Registering allows you to create Dandisets and upload data right away. You can also use the Jupyterhub (<a href="https://hub.dandiarchive.org">https://hub.dandiarchive.org</a>) for computing on dandisets in the cloud. </p>
<p>It may take up to 24 hours for your hub account to be activated and for your email to be registered with our Slack workspace.</p>
<p>Please post any <a href="https://github.com/dandi/helpdesk/discussions">questions</a> or <a href="https://github.com/dandi/helpdesk/issues">issues</a> at our <a href="https://github.com/dandi/helpdesk">Github helpdesk</a>.</p>
<p>Thank you for choosing DANDI for your neurophysiology data needs.</p>
<p>Sincerely,</p>
<p>The DANDI team</p>
```